pkg/collectors/builder_test.go

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package collectors
+
+import (
+	"reflect"
+	"testing"
+
+	"golang.org/x/net/context"
+	koptions "k8s.io/kube-state-metrics/pkg/options"
+	"k8s.io/kube-state-metrics/pkg/whiteblacklist"
+)
+
+var (
+	ctx context.Context = context.TODO()
+)
+
+func TestBuilder_WithApiserver(t *testing.T) {
+	type fields struct {
+		apiserver         string
+		kubeconfig        string
+		namespaces        koptions.NamespaceList
+		ctx               context.Context
+		enabledCollectors []string
+		whiteBlackList    whiteBlackLister
```

Comment:
Rename to allowDenyList.

Reply:
Because the package uses this wording, xavier's change will have to suffice for now. He changed some of the variable names in our code to use allowDenyList.